platform-configs/minnowboard-turbot.robot

```diff
@@ -47,5 +93,5 @@ Power On
     Rte Power Off
```

Comment:
What does `Rte Power Off` do? I can't find the definition in the tree.

Reply:
I don't see any reason why we do Power Off if a couple of lines later we do Relay On. it is pointless.